deploy/chart/templates/0000_50_olm_05-subscription.crd.yaml

```diff
@@ -48,7 +48,12 @@ spec:
           description: Spec for a Subscription
           required:
           - source
+          - sourceNamespace
```

Comment:
These are not actually required fields - could you remove them from the required list?